ui/src/app/shared/test/testsystems/edges.spec.ts

```diff
@@ -392,9 +390,9 @@ export class ems30048 extends AbstractSystem {
 
-  private readonly adminAndInstallerGridModal: FormlyFieldConfig[] = [{ "key": "ems30048", "type": "input", "templateOptions": { "attributes": { "title": "Netz" }, "required": true, "options": [{ "lines": [{ "type": "line", "name": "Keine Netzverbindung!", "channel": "_sum/GridMode" }, { "type": "line", "name": "Bezug", "channel": "_sum/GridActivePower" }, { "type": "line", "name": "Einspeisung", "channel": "_sum/GridActivePower" }, { "type": "line", "indentation": "5%", "channel": "meter0/ActivePowerL1", "children": [{ "type": "line-item", "channel": "meter0/VoltageL1", "indentation": "5%" }, { "type": "line-item", "channel": "meter0/CurrentL1", "indentation": "5%" }, { "type": "line-item", "channel": "meter0/ActivePowerL1", "indentation": "5%" }] }, { "type": "line", "indentation": "5%", "channel": "meter0/ActivePowerL2", "children": [{ "type": "line-item", "channel": "meter0/VoltageL2", "indentation": "5%" }, { "type": "line-item", "channel": "meter0/CurrentL2", "indentation": "5%" }, { "type": "line-item", "channel": "meter0/ActivePowerL2", "indentation": "5%" }] }, { "type": "line", "indentation": "5%", "channel": "meter0/ActivePowerL3", "children": [{ "type": "line-item", "channel": "meter0/VoltageL3", "indentation": "5%" }, { "type": "line-item", "channel": "meter0/CurrentL3", "indentation": "5%" }, { "type": "line-item", "channel": "meter0/ActivePowerL3", "indentation": "5%" }] }, { "type": "line-info", "name": "Die Summe der einzelnen Phasen kann aus technischen Gründen geringfügig von der Gesamtsumme abweichen." }] }] }, "wrappers": ["formly-field-modal"] }];
+  private readonly adminAndInstallerGridModal: OeFormlyView = { "title": "Netz", "lines": [{ "type": "line", "name": "Keine Netzverbindung!", "channel": "_sum/GridMode" }, { "type": "line", "name": "Bezug", "channel": "meter0/ActivePower" }, { "type": "line", "name": "Einspeisung", "channel": "meter0/ActivePower" }, { "type": "line", "indentation": "5%", "channel": "meter0/ActivePowerL1", "children": [{ "type": "line-item", "channel": "meter0/VoltageL1" }, { "type": "line-item", "channel": "meter0/CurrentL1" }, { "type": "line-item", "channel": "meter0/ActivePowerL1" }] }, { "type": "line", "indentation": "5%", "channel": "meter0/ActivePowerL2", "children": [{ "type": "line-item", "channel": "meter0/VoltageL2" }, { "type": "line-item", "channel": "meter0/CurrentL2" }, { "type": "line-item", "channel": "meter0/ActivePowerL2" }] }, { "type": "line", "indentation": "5%", "channel": "meter0/ActivePowerL3", "children": [{ "type": "line-item", "channel": "meter0/VoltageL3" }, { "type": "line-item", "channel": "meter0/CurrentL3" }, { "type": "line-item", "channel": "meter0/ActivePowerL3" }] }, { "type": "line-horizontal" }, { "type": "line-info", "name": "Die Summe der einzelnen Phasen kann aus technischen Gründen geringfügig von der Gesamtsumme abweichen." }] } as OeFormlyView
```

Comment:
Ich würde das JSON trotzdem formatiert abspeichern. So wie es jetzt ist, ist es nicht nachvollziehbar. Können wir diese Variablen vielleicht als statics in eine andere Datei auslagern und importieren?

Reply:
können wir